pallets/order-book/src/types.rs

```diff
@@ -67,6 +67,17 @@ pub enum OrderBookStatus {
     Stop,
 }
 
+#[derive(
+    Encode, Decode, PartialEq, Eq, Copy, Clone, Debug, scale_info::TypeInfo, MaxEncodedLen,
+)]
+pub enum CancelReason {
+    /// User cancels the limit order by themselves
```

Comment:
```suggestion
    /// User cancels the limit order by themself
```
minor correction 🙃

Reply:
Both variant are ok, because of moder notation. But ok